apps/vmq_web_ui/src/vmq_web_ui_cli.erl

```diff
@@ -0,0 +1,106 @@
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(vmq_web_ui_cli).
+-export([register_cli/0]).
+-behaviour(clique_handler).
+
+register_cli() ->
+    clique:register_usage(["vmq-admin", "webui"], webui_usage()),
+    clique:register_usage(["vmq-admin", "webui", "show"], webui_show_usage()),
+    clique:register_usage(["vmq-admin", "webui", "install"], webui_install_usage()),
+    show_cmd(),
+    install_cmd().
+
+show_cmd() ->
+    Cmd = ["vmq-admin", "webui", "show"],
+    Callback =
+        fun
+            (_, [], []) ->
+                Table =
+                    [
+                        [
+                            {'Item', <<"Frontend Path">>},
+                            {'Value', code:priv_dir(vmq_web_ui) ++ "/www"}
+                        ],
+                        [{'Item', <<"Frontend Version">>}, {'Value', "Preview"}]
+                    ],
+                [clique_status:table(Table)];
+            (_, _, _) ->
+                Text = clique_status:text(webui_usage()),
+                [clique_status:alert([Text])]
+        end,
+    clique:register_command(Cmd, [], [], Callback).
+
+install_cmd() ->
```

Comment:
This may not be a good method to ship the WebUI for security reasons, downloading a package from the internet will not be desirable in constrained environments or corporate settings. Instead, having the frontend repo separate and using a precompile hook download the released frontend to the `/priv` directory so the frontend can evolve separately but still ship with VerneMQ releases.